docs/zh/openmldb_sql/task_manage/SHOW_JOBS.md

```diff
@@ -44,6 +57,43 @@ SHOW JOBS;
  1 row in set
 ```
 
+查看指定job id的任务:
+```sql
+SHOW JOBS FROM TASKMANAGER like '1';
+
+---- ------------------ ----------- ------------ --------------- ---------------------------------------------------------------------------------------------------------------------------- --------- ---------------- -------
+  id   job_type           state       start_time   end_time        parameter                                                                                                                    cluster   application_id   error
+ ---- ------------------ ----------- ------------ --------------- ---------------------------------------------------------------------------------------------------------------------------- --------- ---------------- -------
+  1    ImportOnlineData   Submitted   0            1641981373227   LOAD DATA INFILE 'file:///tmp/test.csv' INTO TABLE demo_db.t1 options(format='csv', header=false, mode='append');           local
+ ---- ------------------ ----------- ------------ --------------- ---------------------------------------------------------------------------------------------------------------------------- --------- ---------------- -------
+
+ 1 row in set
+```
+
+### 查看Nameserver里的任务
+查看nameserver里的当前db下的所有任务
+```sql
+show jobs from nameserver;
+ -------- ------------- ------ ------ ----- -------- --------------------- --------------------- ----------
+  job_id   op_type       db     name   pid   status   start_time            end_time              cur_task

```

Comment:
need a extra column as `job_type`, to difference `nameserve` & `taskmanager` from output